tardis/transport/montecarlo/tests/test_r_packet.py

```diff
@@ -10,13 +10,11 @@
 )
 
 
-@pytest.fixture(scope="module")
+@pytest.fixture()
 def simulation_rpacket_tracking_enabled(config_verysimple, atomic_dataset):
```

Comment:
Echo the comments that Josh gave in conftest.py, should double check if the existing modification of config_verysimple in this fixture is an issue or not.

Reply:
Scope doesn't play any role in fixture change. Any scope fixture can change any fixture object. I checked that the fixture with scope function could change the fixture with scope package.

Reply:
I have updated them in both instances.